<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrivKeyDialog</class>
 <widget class="QDialog" name="PrivKeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>248</y>
     <width>480</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>48</y>
     <width>480</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Private Key:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pkField">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>78</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_generateKeysBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>122</y>
     <width>280</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Key</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_loadFromFileBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>169</y>
     <width>280</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load From File</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_errorText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>212</y>
     <width>481</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : #cc0000</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrivKeyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrivKeyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
